interp/value.go

```diff
@@ -350,7 +350,7 @@ func genValueOutput(n *node, t reflect.Type) func(*frame) reflect.Value {
 		}
 		fallthrough
 	case n.anc.kind == returnStmt && n.anc.val.(*node).typ.ret[0].cat == interfaceT:
-		if len(n.anc.val.(*node).typ.ret[0].field) == 0 {
+		if nod, ok := n.anc.val.(*node); !ok || len(nod.typ.ret[0].field) == 0 {
```

Comment:
I understand that we have to check that `n.anc.val.(*node)` is ok to avoid the panic. However, I'm not sur that we want to `return value` when `!ok` (maybe we want to do something else). But oh well, if the tests pass, good enough for now.